woocommerce/woocommerce-ios
# Dynamic Dashboard: Adding Tracks events

Closes: #12558


## Description

This PR adds the tracking events following the tracking plan on pffQ75-dy-p2.

## Testing instructions

- Turn on the feature flag `dynamicDashboard` and build the app.
- On the Dashboard screen, tap the ellipsis menu on any of the card and select the option to hide it. Confirm that event `dynamic_dashboard_hide_card_tapped` is tracked in Xcode console with the correct `type`.
- Tap the Edit button on the top right of Dashboard screen, confirm that event `dynamic_dashboard_edit_layout_button_tapped` is tracked.
- Reorder/enable/disable any item on the list then tap Save. Confirm that event `dynamic_dashboard_editor_save_tapped` is tracked with property `cards` containing only active card names sorted alphabetically.
- Turn off the Internet connection then pull to refresh the dashboard. Then tap on Retry button of any card's error view. Confirm that `dynamic_dashboard_card_retry_tapped` is tracked with the correct `type`.

## Screenshots

N/A

---
